tests/ParserSpecTest.php

```diff
@@ -146,4 +146,35 @@ public function testStringSpecific()
         $parser = new StringParser(new Config(), 'name', 'test');
         $this->assertEquals('test', $parser->defaultsToIfEmpty('default'));
     }
+
+    public function testInRangeValidatorWithValidValues()
+    {
+        $parser = new IntParser(new Config(), 'groupId', 1);
+        $parser->inRange(1, 6);
+        $this->assertEquals(1, $parser->required());
+        $parser = new IntParser(new Config(), 'groupId', 6);
+        $parser->inRange(1, 6);
+        $this->assertEquals(6, $parser->required());
+
+        $parser = new FloatParser(new Config(), 'precipitation', 60.99);
+        $parser->inRange(60.99, 101.12);
+        $this->assertEquals(60.99, $parser->required());
+        $parser = new FloatParser(new Config(), 'precipitation', 101.12);
+        $parser->inRange(60.99, 101.12);
+        $this->assertEquals(101.12, $parser->required());
+    }
+
+    public function testIntInRangeValidatorWithValuesOutOfRange()
+    {
+        $this->setExpectedException(InvalidValueException::class, "Invalid value for parameter \"groupId\". Expected an integer value between 1 and 6, but got \"7\"");
```

Comment:
We could use `'` instead of `"` to avoid the `\"` here :) (e.g. `'invalid value for paramter "groupId"'`)